circle.yml

```diff
@@ -0,0 +1,36 @@
+machine:
```

Comment:
Since this is OSS, we should add it as an OSS repo: https://codeclimate.com/github/signup